docs/operation.md

```diff
@@ -189,43 +190,52 @@ The Anonymization Summary has four statistics:
 
 The median distortion (relative error) is relatively small (less than 2%), but the maximum distortion is high (75%). This is common when there are some bins containing very few protected entities. Again, this may or may not be a data quality problem depending on the use case.
 
 ### Example 3: Bad data quality
 
-![](images/distortion-bad.png#720)
+| Suppressed Count | Suppressed Bins | Median Distortion | Maximum Distortion |
+| --- | --- | --- | --- |
+| 398784 of 440257 (90.58%) | 11933 of 12995 (91.83%) | 94.87% | 97.06% |
 
-The above image illustrates the case when almost all of the data resides in very small bins. 91% of the data is suppressed (`Suppressed Count`), as is 92% of the bins. The median and maximum distortion (relative error) is nearly the same, and both are high (95% and 97%).
+In this case almost all of the data resides in very small bins. 91% of the data is suppressed (`Suppressed Count`), as is 92% of the bins. The median and maximum distortion (relative error) are both high (95% and 97%).
 
 ## Anonymized Data
 
-A toggle switch at the bottom of the Anonymized Data selects between two views. The `Anonymized` view shows only the anonymized data itself: the selected columns and the count. This data is safe to release. The count refers to rows or to protected entities, whichever was chosen in the column selection area.
+A toggle switch at the bottom of the Anonymized Data selects between two views. The `Anonymized view` shows only the anonymized data itself: the selected columns and the count. This data is safe to release. The count refers to rows or to protected entities, whichever was chosen in the column selection area.
 
 The `Combined view` additionally shows the original data side-by-side with the anonymized data. Because the combined view contains original data, it is not safe to release. The combined view presents two additional columns, `Count (original)` and `Distortion`. `Count (original)` is the true count of the original data. `Distortion` is the relative error between the original count and the anonymized count.
 
 The combined view displays suppressed bins. The `Count (anonymized)` column is left blank for suppressed bins.
 
 The combined view lets you examine precisely the distortion and suppression. The anonymized data can be sorted by each column. Sorting by `Count (original)` shows the ascending suppressed bins first, and so is useful for examining what has been suppressed. Sorting by `Distortion` descending shows the bins with the most error first, and so is useful for examining which bins have high distortion.
 
+_Note that the output view displays at most 1000 rows of data._ To view all of the data, use
+the CSV export (Anonymized view only).
```

Comment:
> To view all of the data, use the CSV export (Anonymized view only).

This makes you think that you can only export while in anonymized view. Maybe say (exports anonymized view only)?